<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>fmNotepad</class>
 <widget class="QMainWindow" name="fmNotepad">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>289</width>
    <height>342</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Регистратор</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0" colspan="3">
     <widget class="QLabel" name="label_34">
      <property name="font">
       <font>
        <pointsize>9</pointsize>
       </font>
      </property>
      <property name="lineWidth">
       <number>-5</number>
      </property>
      <property name="text">
       <string>Дата и время</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0" colspan="3">
     <widget class="QLabel" name="lblDateTime">
      <property name="font">
       <font>
        <pointsize>9</pointsize>
       </font>
      </property>
      <property name="lineWidth">
       <number>-5</number>
      </property>
      <property name="text">
       <string>11.11.2011 11-11-11</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0" colspan="5">
     <widget class="QTableWidget" name="tableW">
      <property name="enabled">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <spacer name="horizontalSpacer_2">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>8</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="3" column="1" colspan="2">
     <widget class="QPushButton" name="btnSave">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="text">
       <string>Сохранить</string>
      </property>
     </widget>
    </item>
    <item row="3" column="3" colspan="2">
     <widget class="QPushButton" name="btnClear">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="text">
       <string>Очистить</string>
      </property>
     </widget>
    </item>
    <item row="3" column="5">
     <spacer name="horizontalSpacer_3">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>13</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="4" column="0" colspan="2">
     <spacer name="horizontalSpacer_4">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>49</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="4" column="2">
     <widget class="QLabel" name="label">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="text">
       <string>Интервал сохранения, с</string>
      </property>
     </widget>
    </item>
    <item row="4" column="3">
     <widget class="QSpinBox" name="spinBox">
      <property name="enabled">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="4" column="4">
     <spacer name="horizontalSpacer_5">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>30</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>289</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="muSetting">
    <property name="title">
     <string>Настройка</string>
    </property>
   </widget>
   <widget class="QMenu" name="muAbout">
    <property name="title">
     <string>?</string>
    </property>
   </widget>
   <addaction name="muSetting"/>
   <addaction name="muAbout"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actPortCons">
   <property name="text">
    <string>Консоль порта...</string>
   </property>
  </action>
  <action name="actMBCons">
   <property name="text">
    <string>Консоль драйвера ...</string>
   </property>
  </action>
  <action name="actAD779x">
   <property name="text">
    <string>AD779x...</string>
   </property>
  </action>
  <action name="actParV">
   <property name="text">
    <string>Милливольтметр</string>
   </property>
  </action>
  <action name="actParT">
   <property name="text">
    <string>Термометр</string>
   </property>
  </action>
  <action name="actParCond">
   <property name="text">
    <string>Кондуктометр</string>
   </property>
  </action>
  <action name="actParTP">
   <property name="text">
    <string>Измеритель давления</string>
   </property>
  </action>
  <action name="actParAmper">
   <property name="text">
    <string>Микроамперметр</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
